backend/qemu.pm

```diff
@@ -282,9 +283,10 @@ sub _migrate_to_file {
             execute   => 'migrate-set-parameters',
             arguments => {
                 # This is ignored if the compress capability is not set
-                'compress-level' => $compress_level,
+                'compress-level'   => $compress_level + 0,
```

Comment:
Why are you forcing to integers? Shouldn't block from merging though

Reply:
Because I found that QEMUCPUS was set to a string in one case and then QEMU throws an error because it is expecting a JSON integer, but it got a string. So I decided to just coerce everything to integers.